src/guessTheNumber.js

```diff
@@ -0,0 +1,25 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const generateGuessNumber = Math.floor( Math.random() * 100 ) + 1
+
+const responseTooHigh = 'You answer was higher than'
+const responseTooLow = 'You answer was lower than'
+
+rl.question('Pick a number between 1 and 100\n', answer => {
+  rl.close()
+  if (answer.toString().toLowerCase() === 'exit') {
+    return
+  }
+  answer = Number(answer)
+  const comparison = answer < generateGuessNumber
+    ? 'lower than'
+    : answer > generateGuessNumber
+      ? 'higher than'
+      : 'the same as'
+  console.log(`Your answer is ${comparison} ${generateGuessNumber}`)
```

Comment:
While it wasn't super clear in the puzzle's description, I think that the desired behavior is that the program loops until either a) The user guesses the correct number or b) the user types "Exit"

I challenge you to implement that loop, if you think that focusing on that would benefit you.